### DISCUSSION TITLE
How to make whisper use punctuations more correct?

### DISCUSSION THREAD

**Discussion by zeushera140** [openai/whisper#1936] [Q&A]
For some audios, it uses Punctuations quite good, but for some, it sometimes does not use Punctuations for a long long sentence, especially for some cases it should use period to end this sentence, it did not.

I asked GPT-4 how to make it use Punctuations more correct, there are 2 ways:
1. Use initial prompt, I tried, but failed. It may work for the beginning of audio, but not working well for the whole file.
2. Use gpt to fix the generated result, I have not used this way, but gpt-4 is too expensive for doing this, and gpt-3.5 may be good. However, this way is also a little complex for this job, cause I have to write specific prompt for specific audio.

I also tried whisperx, it tried to end the sentence too often, while usually cut the sentence in the middle.

Any other solutions？

Thanks.

**Comment by glangford** [comment]
See these past discussions, and prefer the use of large-v2 rather than large-v3.

* https://github.com/openai/whisper/discussions/194
* https://github.com/openai/whisper/discussions/194#discussioncomment-3766522

**Comment by Purfview** [comment]
> Any other solutions？

[Standalone Faster-Whisper](https://github.com/Purfview/whisper-standalone-win).

**Reply by zeushera140** [reply]
faster-whisper? I tried, not better.

**Reply by Purfview** [reply]
> faster-whisper?

No. Standalone Faster-Whisper!

**Reply by zeushera140** [reply]
I thought Standalone Faster-Whisper use faster-whisper as backend model. I will try it.

**Comment by sjtu-hxj** [comment]
https://github.com/openai/whisper/discussions/194#discussioncomment-9114184

**Reply by zeushera140** [reply]
thanks, but I tried that, not always work.
